### PR TITLE
Business page

### PR DESCRIPTION
**Pull request by TalReinfeld**
Creating a professional business page skeleton

This PR is depends on PR #127

Solve issue #89 

image

### CODE REVIEW

**Comment by guybeckenstein** [comment]
it failed in flake8 and pytest please fix it

**Review by OfirBachar — COMMENTED** [review]
there are a lot of commit. please squash

File: account/templates/account/business_page.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                    <ul class="social-list">
+                        <li><i class="fa fa-facebook"></i></li>
+                        <li><i class="fa fa-dribbble"></i></li>
+                        <li><i class="fa fa-instagram"></i></li>
+                        <li><i class="fa fa-linkedin"></i></li>
+                        <li><i class="fa fa-google"></i></li>
+                    </ul>
+                    <div class="buttons">
+                        <button class="btn btn-outline-primary px-4">Message</button>
+                        <button class="btn btn-primary px-4 ms-3">Make Appointment</button>
```
Comment (idosinger98) [inline account/templates/account/business_page.html:50]: Please link to the make_appointment I created
{ % url "make_appointment" Professional_id%}

Comment (idosinger98) [reply account/templates/account/business_page.html:50]: Open an issue for that

**Review by idoyek — APPROVED** [review]
LGTM

**Review by Yarboa — APPROVED** [review]
LGTM

**Pull request merged**